### ISSUE TITLE
HTML-Proofer found 2 failures

### ISSUE THREAD

**Issue by ajsb85** [flyve-mdm/android-mdm-agent#464]
```bash
- ./_site/screenshots.html
  *  External link https://github.com/flyve-mdm/flyve-mdm-android-agent/compare/v0.99.45...v2.0.0 failed: 404 No error
  *  External link https://github.com/flyve-mdm/flyve-mdm-android/compare/v0.99.45...v0.99.46 failed: 404 No error
htmlproofer 3.7.2 | Error:  HTML-Proofer found 2 failures!
```

**Comment by Naylin15** [comment]
This issue will be solved along with the PR #466 and #457

**Comment by Naylin15** [comment]
The failures regarding these links were fixed

```
- ./_site/screenshots.html
  *  External link https://github.com/flyve-mdm/flyve-mdm-android-agent/compare/v0.99.45...v2.0.0 failed: 404 No error
  *  External link https://github.com/flyve-mdm/flyve-mdm-android/compare/v0.99.45...v0.99.46 failed: 404 No error
htmlproofer 3.7.2 | Error:  HTML-Proofer found 2 failures!
```

However now there are new failures of broken links

**Comment by Naylin15** [comment]
I've been making tests on my fork, I found that some links are already been generated with the failure but html proofer detects them when adding the header for the project site

- [Test adding the headers locally](https://circleci.com/gh/Naylin15/android-mdm-agent-fork/1164)

**Comment by Naylin15** [comment]
Making tests in my fork to solve this issue, I have a solution for the failures caused by renaming the overview-summary.html to index.html 

By updating the links remaining, 407 failures are fixed

![captura de pantalla 2018-06-15 a la s 5 42 19 p m](https://user-images.githubusercontent.com/30867977/41490761-7fcee698-70c3-11e8-81a0-06a023e1498d.png)

This change might be made with a script in CI

**Comment by Naylin15** [comment]
Most of the failures will be fixed in #484 this regarding only the broken links left from the renaming of files

However there are 50 failures that comes from the generation of the documentation itself
